Split the "work pools and workers" page into two pages (#14298)
PrefectHQ/prefect: docs/3.0rc/deploy/control-runs/prefect-deploy.mdx

---

In this guide, you will configure a deployment that uses a work pool for dynamically provisioned infrastructure.
Work pools and workers bridge the Prefect _orchestration environment_ with your _execution environment_. 
When a [deployment](/3.0rc/deploy/serve-flows/) creates a flow run, it is submitted to a specific work pool for scheduling. 

> A worker running in the execution environment polls its respective work pool for new runs to execute; or the work pool submits flow runs to serverless infrastructure directly, depending on your configuration.

Reviewer: ```suggestion
A worker running in the execution environment polls its respective work pool for new runs to execute; or the work pool submits flow runs to serverless infrastructure directly, depending on the type of work pool.
```
